core/geometry/src/polyface/IndexedPolyfaceVisitor.ts

```diff
@@ -202,49 +201,53 @@ export class IndexedPolyfaceVisitor extends PolyfaceData implements PolyfaceVisi
       this.color.push(Geometry.interpolateColor(other.color[index0], fraction, other.color[index1]));
     // TODO: auxData? taggedNumericData?
   }
+  /** Create a visitor for a subset of the facets visitable by the instance. */
+  public createSubsetVisitor(facetIndices: number[], numWrap: number = 0): IndexedPolyfaceSubsetVisitor {
+    return IndexedPolyfaceSubsetVisitor.createSubsetVisitor(this._polyface, facetIndices, numWrap);
+  }
 }
+
 /**
  * An `IndexedPolyfaceSubsetVisitor` is an `IndexedPolyfaceVisitor` which only visits a subset of facets in the polyface.
  * * The subset is defined by an array of facet indices provided when this visitor is created.
  * * Input indices (e.g., for `moveToReadIndex`) are understood to be indices into the subset array.
  * @public
  */
 export class IndexedPolyfaceSubsetVisitor extends IndexedPolyfaceVisitor {
-  private _parentFacetIndices?: number[]; // only undefined during super constructor!
-  private _currentActiveIndex: number;    // index within _parentFacetIndices, or -1 after construction
-  private _nextActiveIndex: number;       // index within _parentFacetIndices
+  private _facetIndices: number[];
+  private _currentSubsetIndex: number;    // index within _facetIndices, or -1 after construction
```

Comment:
I think the space was there to align with previous comment but now you removed the previous comment so you can remove space as well

```suggestion
  private _currentSubsetIndex: number; // index within _facetIndices, or -1 after construction
```